charts/validator/README.md

```diff
@@ -67,3 +67,5 @@ A Helm chart for deploying Chronicle Validator on Kubernetes
 | tor-proxy | object | `{"enabled":true}` | Values for Tor Proxy (subchart of ghost) |
 | tor-proxy.enabled | bool | `true` | values for tor-proxy, installs [tor-controller](/crds/tor-controller.yaml) and creates an [onionService CRD](/templates/tor-onion-service.yaml) |
 
+----------------------------------------------
```

Comment:
how about we agree not to use the footer?

Reply:
this is auto generated. It stays

Reply:
it is not generated for me and also it is not consistent

Reply:
we'll fix that for you outside of this